janitor/functions/expand_grid.py

```diff
@@ -36,41 +36,43 @@ def expand_grid(
     `others`, the columns are flattened, before the final
     cartesian DataFrame is generated.
 
-    If a Pandas Series/DataFrame is passed, and has a labeled index, or
+    If a pandas Series/DataFrame is passed, and has a labeled index, or
     a MultiIndex index, the index is discarded; the final DataFrame
     will have a RangeIndex.
 
     The MultiIndexed DataFrame can be flattened using pyjanitor's
-    `collapse_levels` method; the user can also decide to drop any of the
-    levels, via Pandas' `droplevel` method.
-
-    Functional usage syntax:
-
-    ```python
-
-        import pandas as pd
-        import janitor as jn
-
-        df = pd.DataFrame(...)
-        df = jn.expand_grid(df=df, df_key="...", others={...})
-    ```
-
-    Method-chaining usage syntax:
-
-    ```python
-        import pandas as pd
-        import janitor as jn
-
-        df = pd.DataFrame(...).expand_grid(df_key="bla",others={...})
-    ```
-
-    Usage independent of a DataFrame
-
-    ```python
-        import pandas as pd
-        from janitor import expand_grid
-
-        df = expand_grid(others = {"x":range(1,4), "y":[1,2]})
+    [`collapse_levels`][janitor.functions.collapse_levels.collapse_levels]
+    method; the user can also decide to drop any of the levels, via Pandas'
+    `droplevel` method.
+
+    Example:
+
+        >>> import pandas as pd
+        >>> import janitor as jn
+        >>> df = pd.DataFrame({"x": [1, 2], "y": [2, 1]})
+        >>> data = {"z": [1, 2, 3]}
+        >>> df.expand_grid(df_key="df", others=data)
+          df     z
+           x  y  0
+        0  1  2  1
+        1  1  2  2
+        2  1  2  3
+        3  2  1  1
+        4  2  1  2
+        5  2  1  3
+
+    Expand_grid works with non-pandas objects:
+
+        >>> data = {"x": [1, 2, 3], "y": [1, 2]}
+        >>> jn.expand_grid(others=data)
+           x  y
+           0  0
+        0  1  1
+        1  1  2
+        2  2  1
+        3  2  2
+        4  3  1
+        5  3  2
 
```

Comment:
`expand_grid` --- Could you also help to sneak in a quick fix of the triple-quoted error message problem (about 25 lines below this line)?